schematic/utils/csv_utils.py

```diff
@@ -239,9 +239,9 @@ def create_schema_classes(schema_extension: pd.DataFrame, se: SchemaExplorer) ->
             for p in props:
                 prop_2_class[p.strip()] = record["Attribute"]
 
-    print("=====================")
-    print("Adding attributes")
-    print("=====================")
+    logger.info("=====================")
```

Comment:
Again, I think these lines clutter the output.

Reply:
let's remove :)